### ISSUE TITLE
Background colour of the editor is changed

### ISSUE THREAD

**Issue by grcq** [samdark/intellij-visual-studio-code-dark-plus#60]
### What the issue is?
The background colour of the editor is suddenly white. It wasn't like this before. I had an outdated version of IntelliJ, and I just updated it. I have absolutely no idea how this happened, but I cannot remember it like this.

### Screenshots
![Skjermbilde 2023-08-08 230454](https://github.com/samdark/intellij-visual-studio-code-dark-plus/assets/66485591/97ef9954-c98a-42d9-b75c-d300bfc774ad)

### Additional info

| Q                | A
| ---------------- | ---
| Plugin Version   | 2.8
| IDE and version  |  IntelliJ 2023.2
| Operating system | Windows

**Comment by samdark** [comment]
Wow. That doesn't look good. Can you look under editor color schemas and check if proper theme is selected?

**Comment by jsodeman** [comment]
I saw the same thing in WebStorm and tried toggling the theme back and forth, restarting the IDE, rebooting Windows. Rider looks fine though, both being 2023.2 and with the updated and migrated JetBrains Toolbox.

When I checked under Editor > Color Scheme the Scheme was set to Classic Light for some reason. I changed it to Visual Studio Code Dark Plus and now the theme is displaying correctly.

![image](https://github.com/samdark/intellij-visual-studio-code-dark-plus/assets/2388842/593fcf03-753b-4ef1-9870-1fcded39a497)

![image](https://github.com/samdark/intellij-visual-studio-code-dark-plus/assets/2388842/94256695-ca37-46a2-b82d-c99ebf6ed884)
